pages/common/nvm.md

```diff
@@ -1,28 +1,33 @@
 # nvm
 
-> Node.js version manager.
-> Switch between NodeJS versions: system, node, 0.10, 0.12, 4.2 etc.
+> Install, uninstall or switch between Node.js versions.
+> Supports version numbers like "0.12" or "v4.2", and labels like "stable", "system", etc.
+> Homepage: <https://github.com/creationix/nvm>.
 
-- Install a specific version of NodeJS:
+- Install a specific version of Node.js:
```

Comment:
Do all the pages mention Node.js instead of NodeJS ?

Reply:
I didn't check that. This one in particular used both spellings, but the latter is the official one, according to the Node.js website and Wikipedia.

I just checked, and only svgo.md uses the "Nodejs" spelling. Since it's such a  minor additional change, I guess we can include it in this PR.

Reply:
Huh, interesting! The [official website](https://nodejs.org/) uses the spelling `Node.js`. I've sat an exam today and used `Node.JS` instead - I'll have to remember this!